lookout/style/typos/train_test_corrector.py

```diff
@@ -0,0 +1,242 @@
+from typing import Tuple, Union
+
+from google_drive_downloader import GoogleDriveDownloader as gdd
+import pandas
+
+from lookout.style.typos.corrector import TyposCorrector
+from lookout.style.typos.preprocessing.create_typos import corrupt_tokens_in_df, train_test_split
+from lookout.style.typos.preprocessing.filter_identifiers import filter_splitted_identifiers
+from lookout.style.typos.preprocessing.metrics import print_scores
+from lookout.style.typos.preprocessing.pick_subset import pick_subset_of_df
+from lookout.style.typos.preprocessing.print_frequencies import print_frequencies
+from lookout.style.typos.utils import COLUMNS, flatten_df_by_column
+
+
+DEFAULT_VOCABULARY_SIZE = 10000
+DEFAULT_VOCABULARY_PATH = "lookout/style/typos/data/vocabulary.csv"
+DEFAULT_FREQUENCY_PATH = "lookout/style/typos/data/frequencies.csv"
+
+DRIVE_DATASET_ID = "1muNVWPe68XK8SFvqIv3V728NmkT46aTx"
+DRIVE_FASTTEXT_ID = "1hCOIwKn-QZLVv1S385HxyNMeERgKGIvo"
```

Comment:
Fasttext is a model => it is stored in Modelforge => we should use Modelforge to fetch it.